### ISSUE TITLE
Adding CenterNet to Torchvision/detection

### ISSUE THREAD

**Issue by chang4869** [pytorch/vision#4595]
### 🚀 The feature

Add CenterNet to torchvision

### Motivation, pitch

centernet is a better one-stage object detector than SSD and it is also very practical.This anchor-free detector is currently relatively popular.

**Comment by mikeseven** [comment]
There are 2 centernet in the literature. 
The most used I think is [centernet objects as point](https://github.com/xingyizhou/CenterNet) that is the basis for many applications.

The nice thing about this model is that it is a generic architecture and many other models already in torchvision can be used for backbone, neck, and head.

**Comment by adamsau** [comment]
@datumbox is there any plan on adding this new model?

**Comment by datumbox** [comment]
@adamsau Thanks for checking.

Our intention is to prioritize the model on the future but not yet. The main issue at the moment is that we don't want to add more Keypoint detection models (I'm aware CenterNet supports normal bboxes too) until we have completed the [Transforms v2 API](https://github.com/pytorch/vision/tree/main/torchvision/prototype/transforms). This is because in order to train the models properly, we need to have better support on the transforms side for Keypoints.

**Comment by adamsau** [comment]
@datumbox thanks for the update, definitely need some fast model to detect keypoints since the keypoint rcnn is too slow for real world applications. Hope there will be new models coming soon!

**Comment by datumbox** [comment]
Makes sense. Can certainly ping you once we are ready in case you are willing to contribute it. :)
